packages/compiler-dom/src/transforms/stringifyStatic.ts

```diff
@@ -356,7 +358,7 @@ function stringifyElement(
 // (see compiler-core/src/transforms/transformExpression)
 function evaluateConstant(exp: ExpressionNode): string {
   if (exp.type === NodeTypes.SIMPLE_EXPRESSION) {
-    return new Function(`return ${exp.content}`)()
+    return new Function(`return ${removeEscapes(exp.content)}`)()
```

Comment:
I don't think removing all of these characters is safe, as it'll also remove them from inside the expression. e.g. ``v-text="`text1\\text2`"`` will render as `text1text2` ([example](https://deploy-preview-7995--vue-sfc-playground.netlify.app/#eNqtUctOwzAQ/BXLZ2yLl1CjgMR/+NCQbMHIj5W9SYmq/DubpC03emh98ezseNaaPch3RD30ICtZlzY7JFGAenyzsTYrwZALgoC+IeBKiLpzwwIYJi9a35TyamUY1QKtPDa57d0JCjEogh9i4Xa+7y2fGTxsrTyJzIWXf5QQiws/PlNX2DzexubpX5vaJL/mZ9YAa3OOVd5JFzBlUqFB/V1S5JUcZrE9NjjWSizMzPHO5trKLyIslTEdoE+jwgyDg7162WyelWKVKrtW8YzxM6c+djoCebcbdYNouK1zH8kF0FCC+shpXyDz+PXzk42TnH4B8FSgew==)).

Is the problem that the expression ends up on a separate line, so the `return` just ignores it? If so, then wouldn't just trimming `exp.content` be enough? Or wrapping it in parentheses to avoid the early return?

Reply:
cc 👀